modules/cloudsql/main.tf

```diff
@@ -12,6 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+data "google_compute_network" "compute_network" {
+  name = var.private_network
+}
+
+resource "google_compute_global_address" "private_ip_address" {
+  name          = "private-ip-address"
+  purpose       = "VPC_PEERING"
+  address_type  = "INTERNAL"
+  prefix_length = 16
+  network       = data.google_compute_network.compute_network.id
+}
+
+resource "google_service_networking_connection" "private_vpc_connection" {
```

Comment:
we will move this out to infrastructure module.

Reply:
Hmm i am not sure, this is for CloudSQL setup only, so it is better to bundle with CloudSQL.

Reply:
The infra set up does not have google_service_networking_connection setup though? The problem is cloudsql private IP requires this to exist.